docs/plugins/private-plugins.md

```diff
@@ -0,0 +1,268 @@
+---
+id: private-plugins
+slug: /private-plugins
+toc_max_heading_level: 2
+---
+
+# Creating and Using Private Plugins
```

Comment:
In some places (in the docs) it is called "custom plugins" and here it's "private plugins". I would go with private plugins all over the place